frontend/talentsearch/src/js/components/applicantProfile/ProfileFormWrapper.tsx

```diff
@@ -20,20 +21,24 @@ const ProfileFormWrapper: React.FunctionComponent<ProfileFormWrapperProps> = ({
   title,
   cancelLink,
   children,
+  prefixBreadcrumbs = true,
 }) => {
   const intl = useIntl();
   const profilePath = useApplicantProfileRoutes();
-  const links = [
-    {
-      title: intl.formatMessage({
-        defaultMessage: "My Profile",
-        description: "Breadcrumb from applicant profile wrapper.",
-      }),
-      href: profilePath.myProfile(),
-      icon: <UserIcon style={{ width: "1rem", marginRight: "5px" }} />,
-    },
-    ...crumbs,
-  ];
+  let links = [...crumbs];
+  if (prefixBreadcrumbs) {
+    links = [
+      {
+        title: intl.formatMessage({
+          defaultMessage: "My Profile",
+          description: "Breadcrumb from applicant profile wrapper.",
+        }),
+        href: profilePath.myProfile(),
+        icon: <UserIcon style={{ width: "1rem", marginRight: "5px" }} />,
```

Comment:
The `icon` key is no longer used in the [Breadcrumbs component](https://github.com/GCTC-NTGC/gc-digital-talent/commit/fc457b19722bf41493d4303eb4405233064ed4e1), though strangely, still [part of the props](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/frontend/common/src/components/Breadcrumbs/Breadcrumbs.tsx#L6).

Reply:
Removed in [remove icons from breadcrumbs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3669/commits/c62df89324658bcdbeb127504a564c6ce20e83db)

Reply:
The icons still remain in some components and stories for Breadcrumbs:

![Screen Shot 2022-08-24 at 12 26 44](https://user-images.githubusercontent.com/3046459/186472637-2b7d08ad-f598-4499-826f-26af4ba420be.png)

Reply:
Removed all in [remove icons from breadcrumbs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3669/commits/dba59efa6059435cf6640f3def0f5fb8082c524d)